samples/bluetooth/cap_initiator/prj.conf

```diff
@@ -21,3 +21,4 @@ CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT=2
 # Broadcast sources values if enabled by CONFIG_SAMPLE_BROADCAST
 CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT=2
 CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
+CONFIG_BROADCAST_ID=0x123456
```

Comment:
Did you consider defaulting `BROADCAST_ID` to `0x123456`? 

Or just leave it to default to `0x000000`?

Reply:
Actually, I intentially chose to not default it, forcing the user to set it.

Reply:
> Actually, I intentially chose to not default it, forcing the user to set it.

The Kconfig defaults to `0`. You cannot "not default" a Kconfig option :)